lib/miniproxy/server.rb

```diff
@@ -10,6 +10,14 @@ module MiniProxy
   class Server
     DRB_SERVICE_TIMEOUT = 5
 
+    def self.get_config(k)
```

Comment:
What about just providing a `allow_external_requests` getter/setter which does this for you? What is the advantage of providing methods to change arbitrary config values? For example, it would be nicer to be able to write:

    MiniProxy::Server.allow_external_requests = true

Reply:
I agree, I feel like explicit methods are clearer from a self documenting point of view.